slangpy/tests/slangpy_tests/test_torch_dtypes.py

```
@@ -0,0 +1,101 @@
+    func = create_function_from_module(
+        device, "add_d", "double add_d(double a, double b) { return a + b; }"
+    )
+    a = torch.tensor([1.5, 3.0], device="cuda", dtype=torch.float64)
+    b = torch.tensor([2.5, 4.0], device="cuda", dtype=torch.float64)
+    result = func(a, b)
+    assert result.dtype == torch.float64
+    assert torch.allclose(result, torch.tensor([4.0, 7.0], device="cuda", dtype=torch.float64))
+
+
+@pytest.mark.parametrize(
+    "slang_func,torch_dtype",
```

Ruff (0.16.0)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)